keps/NNNN-kep-template/README.md

```diff
@@ -115,11 +122,15 @@ Check these off as they are completed for the Release Team to track. These
 checklist items _must_ be updated for the enhancement to be released.
 -->
 
-- [ ] Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
-- [ ] KEP approvers have approved the KEP status as `implementable`
-- [ ] Design details are appropriately documented
-- [ ] Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
-- [ ] Graduation criteria is in place
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] Production readiness review approved
 - [ ] "Implementation History" section is up-to-date for milestone
 - [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
```

Comment:
Are there features where it makes sense to review whether the documentation is good enough, and to have those “good enough” criteria agreed before targeting to a release?

Thinking from the SIG Docs side, there's a range of experience, from features that are well documented in beta and where graduation is a formality, to features that are targeted to go GA in the next release and then there's a bit of a scramble to complete the documentation. And there are also things that go GA and _then_ get properly documented.

Reply:
That makes sense to me, but it's orthogonal to PRR effort. I think you should open a separate PR for it and open it for discussion.